Makefile

```diff
@@ -46,46 +49,30 @@ apply-migrations: ensure-venv
 reset-db: drop-db create-db apply-migrations
 
 setup-pyenv:
-	./scripts/pyenv_setup.sh
+	@./scripts/pyenv_setup.sh
 
 ensure-venv:
-	./scripts/ensure-venv.sh
-
-ensure-pinned-pip: ensure-venv upgrade-pip
+	@./scripts/ensure-venv.sh
 
 upgrade-pip:
-	./scripts/python.sh upgrade-pip
+	@./scripts/do.sh upgrade-pip
 
 setup-git-config:
-	@git config --local branch.autosetuprebase always
-	@git config --local core.ignorecase false
-	@git config --local blame.ignoreRevsFile .git-blame-ignore-revs
+	@./scripts/do.sh setup-git-config
 
 setup-git: ensure-venv setup-git-config
```

Comment:
At this point, I'd just move these prerequisites to lib.sh. Technically in make-land, targets aren't supposed to be ordered.

Reply:
Just do this for anything that involves do.sh.